Idno/start.php

```diff
@@ -94,9 +94,15 @@ function &loader()
 
 // Bonita is being used for templating
     $known_loader->registerNamespace('Bonita', dirname(dirname(__FILE__)) . '/external/bonita/includes');
+//PSR Log Interface    
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LoggerInterface.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/AbstractLogger.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LogLevel.php';
+
```

Comment:
If you clone https://github.com/php-fig/log into external/, then you should be able to call
```php
registerNamespace('Psr\Log', __DIR__.'/external/log');
```
I think the issue right now is that you have the Psr namespace right in external rather than a subdirectory like the other external packages